CMakeLists.txt

```diff
@@ -610,3 +610,10 @@ add_subdirectory (utils)
 include (cmake/print_include_directories.cmake)
 
 include (cmake/sanitize_target_link_libraries.cmake)
+
+
+add_executable (meili   src/Storages/MeiliSearch/meili.cpp 
```

Comment:
This is incorrect. Should be added like here https://github.com/ClickHouse/ClickHouse/blob/master/src/CMakeLists.txt#L101